docker/scripts/hardware/component/disk.py

```diff
@@ -24,6 +24,9 @@ def __init__(self, lsblk):
         if not self.__is_nvme():
             self.data["smart"] = utils.get_smart_attributes(self.lsblk["name"])
 
+        if self.__is_nvme():
```

Comment:
Given the two conditions involved, perhaps this and `if` above it could be converted to an `if`/`else`?